plugins/resource_management/app/controllers/resource_management/domain_admin_controller.rb

```diff
@@ -68,6 +70,55 @@ def update
       end
     end
 
+    def reduce_quota
+      # Note: this function is used in two ways
+      # first show the reduce quota window
+      # second: reduce the quota
```

Comment:
Ah, that's the source for my other confusion down below. Can you split that into two actions, please?